fala/playwright/setup.py

```diff
@@ -47,7 +47,7 @@ def visit_results_page(self, **kwargs):
         for label in checkbox_labels:
             page.get_by_label(label).check()
         page.get_by_role("button", name="Search").click()
-        if page.locator("#changeSearchButton"):
+        if page.locator("#changeSearchButton").count() > 0:
```

Comment:
I think you can use the following:

`if page.locator("#changeSearchButton").is_visible():` instead of count. Count does not feel right. Give it a try but I'm pretty checking visibility over counting is the preferred way to assert.

Reply:
👍 amended to use is_visible() as suggested.